lib/ansible/modules/packaging/os/opkg.py

```diff
@@ -148,12 +157,12 @@ def install_packages(module, opkg_path, packages):
     install_c = 0
 
     for package in packages:
-        if query_package(module, opkg_path, package):
+        if query_package(module, opkg_path, package, latest):
             continue
 
         rc, out, err = module.run_command("%s install %s %s" % (opkg_path, force, package))
```

Comment:
Does `opkg install` actually upgrade the package if it needs to? [Docs](https://openwrt.org/docs/guide-user/additional-software/opkg) seem to say there's an `upgrade` subcommand?

Reply:
I did some testing but the behavior seems to be the same. Both `install` and `upgrade` install the package missing and upgrade the package if an upgrade is available (if the repos where updated previously).

Reply:
@dangowrt right?

Reply:
Alright we'll trust your tests at this point. Thanks!